### PR TITLE
Convert alias to query param

### PR DESCRIPTION
**Pull request by nikhilvenkatraman-okta**
Resolves: OKTA-359902

- Convert alias id to be a query param 
- Refactored some code/tests 
@okta/eng-uicore  @MarkCowan-okta

### CODE REVIEW

File: i18n-util/readme.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+
+Admin links will get redirected to en version of the docs since we don’t have translations yet.
+Example:
+
+http://help.okta.com/okta_help.htm?id=csh-config-asso
+
+End-user links will get redirected to the localized version of the docs since we have translations.
+Example:
+
+https://help.okta.com/okta_help.htm?id=csh_ov_overview?type=end-user
```
Comment (ganeshsomasundaram-okta) [inline i18n-util/readme.md:15]: nit: can we update this to &
`https://help.okta.com/okta_help.htm?id=csh_ov_overview**?**type=end-user`

File: i18n-util/readme.md [hunk 2]
```diff
@@ -38,7 +55,7 @@ Any time we make changes to src files within `i18n-util`, we need to make sure w
 
   Locale is computed within the LocaleUtil based on a priority list where we look at locale in the following order. A similar priority is used while computing locale for the okta monolith app.
 
- 1. Locale in query param Example help.okta.com/okta_help.htm#csh-eu-quick-access-apps?locale=ja
+ 1. Locale in query param Example help.okta.com/okta_help.htm?id=csh-eu-quick-access-apps?locale=ja
```
Comment (ganeshsomasundaram-okta) [inline i18n-util/readme.md:58]: nit: same as above for the "?"

File: i18n-util/readme.md (hunk 1 above)
Comment (nikhilvenkatraman-okta) [reply i18n-util/readme.md:15]: yes, good catch

File: i18n-util/src/UrlUtil.js [hunk 3]
```diff
@@ -1,49 +1,34 @@
...
-  const host = aliasUrl.split('/okta_help.htm')[0];
+  const host = window.location.href.split('/okta_help.htm')[0];
   // Currently type is only used for end-user docs. In the future it will be used for admin docs
   const type = getUrlParameter('type');
-  let cshid = aliasUrl.split('okta_help.htm')[1];
-  // Remove locale form url
-  cshid = cshid.split('?locale=')[0];
-  // Remove type form url
-  cshid = cshid.split('?type')[0];
+  const id = getUrlParameter('id');
```
Comment (ganeshsomasundaram-okta) [inline i18n-util/src/UrlUtil.js:18]: @nikhilvenkatraman-okta just wondering if we have to validate this for any potential XSS attacks by using the query params. I wanted to run it by you just to be safe.

Comment (nikhilvenkatraman-okta) [reply i18n-util/src/UrlUtil.js:18]: @ganeshsomasundaram-okta 
`id` or any of the other parameters are not something that is submitted by a user from a form. All these parameters are coming from help links that are constructed within the monolith.

File: i18n-util/readme.md (hunk 2 above)
Comment (nikhilvenkatraman-okta) [reply i18n-util/readme.md]: Fixed

**Review by ganeshsomasundaram-okta — APPROVED** [review]
lgtm!

**Pull request merged**